runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaClientProduceFactory.java

```diff
@@ -583,6 +591,26 @@ private int flushRecordContFin(
         return progress;
     }
 
+    private int rejectAll(
+        KafkaProduceStream.KafkaProduceClient client,
+        long traceId,
+        long authorization,
+        long budgetId,
+        int reserved,
+        int flags,
+        OctetsFW payload,
+        OctetsFW extension,
+        int progress,
+        int limit)
+    {
+        client.cleanupNetwork(traceId);
```

Comment:
Can we move this to the place where we assign flusher to `flushRecordIgnoreAll` instead?